train model.py

```diff
@@ -0,0 +1,93 @@
+from model import CNNtoRNN
```

Comment:
Please rename file (no spaces in name)

also add first shebang line `#/usr/bin/env python3`

you can also make this file executable (`chmod +x <filename>` if you are using Linux)